listenbrainz/bigquery-writer/bigquery-writer.py

```diff
@@ -81,7 +81,7 @@ def callback(self, ch, method, body):
             }
             bq_data.append({
                 'json': row,
-                'insertId': "%s-%s" % (listen['user_name'], listen['listened_at'])
+                'insertId': "%s-%s-%s" % (listen['user_name'], listen['listened_at'], listen['recording_msid'])
```

Comment:
@alastair, because the `insertId` for two listens with different metadata and same ts was the same, only one of them would get written into BQ. A question is what would be the ideal way to write tests for stuff like this, so that this doesn't break again?